### PR TITLE
Crowbar variants

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Crowbars are now unable to be fit into pockets, and can additionally be placed on your belt slot. There are now new "variants" of crowbars that can be found around the station from different sources:
- Plain crowbars can be found in tool storage and maintenance.
- Green crowbars can be obtained through lathes.
- Red crowbars can be found in emergency lockers.
- Yellow crowbars can be found in engineering vendors.
- Orange crowbars can be found in toolboxes.

_I also tweaked the emergency crowbar sprite to have proper outlines._

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

**Changelog**

:cl: Ubaser
- tweak: Normal crowbars cannot be placed in pockets, but can now fit in belts.
- tweak: Depending on where you obtained the crowbars, they will now have different colours.

### CODE REVIEW

**Comment by UbaserB** [comment]
FYI I'm open to any balance changes, what I'm looking for here is some kind of gameplay difference between the emergency crowbar and normal crowbar otherwise one should be removed imo.
My goal here was to make the normal crowbar more useful to tiders and engineering (maybe science too), while most people will ordinarily take the emergency crowbar just for prying open doors or firelocks.

**Comment by forgotmyotheraccount** [comment]
I feel that the emergency crowbar sprite should be visibly smaller if the standard crowbar can no longer fit in pockets

**Comment by LittleNyanCat** [comment]
You should probably mention in the changelog that these changes don't affect emergency crowbars. Simply saying "crowbars" will lead most people to assume you're talking about both regular and emergency crowbars.

**Comment by Redfire1331** [comment]
you should make the sprite for the emergency crowbar smaller to accommodate the change

**Comment by MACMAN2003** [comment]
I feel like this would make the regular crowbar a direct downgrade to the emergency crowbar because it can't be placed in the pockets

**Comment by deltanedas** [comment]
you can still presumably put it in a toolbelt

**Comment by Partmedia** [comment]
I'll prefix this by saying this is a reasonable differentiation that can be considered.

I do want to say that what makes an "emergency crowbar" different from a "crowbar" _is_ the "skin". You can see real-life analogs in things like "emergency flashlights" and "emergency life preservers". They are not functionally different from the normal item. They're painted a different color so that if somebody nabs them in a non-emergency situation they can be yelled at/publicly shamed/arrested.

So even if there were to be no differentiation, then the differentiation is perhaps that you obtained the item from looting an emergency cabinet, and should be disciplined for stealing a safety item when there was no emergency.

**Comment by UbaserB** [comment]
That makes a lot of sense, but if you also think about some "emergency" items, they are designed differently (often more compact) for the purpose they are intended for.

**Comment by LankLTE** [comment]
I'm kind of with a few other people that the normal crowbar feel flat-out worse after this? Crowbars still aren't a viable weapon even while wielded, and they're mostly utility items, so the more portable one is just going to be better.

**Comment by Emisse** [comment]
they shouldnt be different

**Comment by Djungelskog2** [comment]
> they shouldnt be different

I completely agree

**Comment by UbaserB** [comment]
The PR is now about crowbar variants and tweaking how it's stored (no longer in pocket but for belts)

**Comment by SlamBamActionman** [comment]
No blue/purple crowbar? But my rainbow... :(

**Comment by Dutch-VanDerLinde** [comment]
pocket crowbar in you tool and engineering, emergency crowbar is big?

**Comment by UbaserB** [comment]
> pocket crowbar in you tool and engineering, emergency crowbar is big?

they're all basically normal crowbars now

**Comment by Dutch-VanDerLinde** [comment]
> > pocket crowbar in you tool and engineering, emergency crowbar is big?
> 
> they're all basically normal crowbars now

ik im making a suggestion

**Comment by lzk228** [comment]
you forgot the Why part in your pr description

**Comment by UbaserB** [comment]
> you forgot the Why part in your pr description

im just waiting for emisse to merge it because it was their decision

**Comment by ninruB** [comment]
I'd expect some justification for the changes because it comes off as an overall nerf to all crowbars.

**Comment by UbaserB** [comment]
> I'd expect some justification for the changes because it comes off as an overall nerf to all crowbars.

https://discord.com/channels/310555209753690112/770682801607278632/1251414397478502461

**Comment by ninruB** [comment]
> https://discord.com/channels/310555209753690112/770682801607278632/1251414397478502461

Unless I can't read, this doesn't provide any justification except for "it should be this way".

**Comment by UbaserB** [comment]
> Unless I can't read, this doesn't provide any justification except for "it should be this way".

It's dumb that you can fit a giant crowbar in your pocket

**Comment by ninruB** [comment]
> It's dumb that you can fit a giant crowbar in your pocket

This implies that the crowbar is giant, that it doesn't hang off the pocket, and that changes surrounding videogame balance should be based on immersion/realism. I personally see no issue with having crowbars in pockets from a balance perspective.

**Comment by UbaserB** [comment]
> > It's dumb that you can fit a giant crowbar in your pocket
> 
> This implies that the crowbar is giant, that it doesn't hang off the pocket, and that changes surrounding videogame balance should be based on immersion/realism. I personally see no issue with having crowbars in pockets from a balance perspective.

look ive had a stroke trying to balance this pr and ive given up on thinking about it any further because what the maintainers want was way different than my idea and its not like im going to keep arguing with them

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:48]: why dont write `state: icon`, size and shape in BaseCrowbar?

Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:48]: oh thats true, but i will still have to keep the state: icon because of the rsi change for each crowbar i believe

Comment (lzk228) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:48]: > oh thats true, but i will still have to keep the state: icon because of the rsi change for each crowbar i believe

state icon also is taken from parent

**Comment by forgotmyotheraccount** [comment]
So just to be clear emergency crowbars can still be placed in pockets? These color variants sound like they'd be a big headache for new players, as I said the emergency crowbar should really have a new smaller sprite if it's going to have different properties to all the other different crowbars

**Comment by UbaserB** [comment]
all crowbars are the same

**Comment by Hanzdegloker** [comment]
With that many colours, wouldn't it be better to have a grey scale sprite overlay and just RGB colour it instead so many new sprites that are so close in looks. Like how the casual jumpsuits has 3 colour variants but recolour a greyscale sprite to save space.

**Comment by Emisse** [comment]
> With that many colours, wouldn't it be better to have a grey scale sprite overlay and just RGB colour it instead so many new sprites that are so close in looks. Like how the casual jumpsuits has 3 colour variants but recolour a greyscale sprite to save space.

yea do this

**Comment by Flareguy** [comment]
This is minor and not that important, but you should probably prefix your RSIs like ``CrowbarColor`` instead of ``ColorCrowbar``, so that they appear together when sorting alphabetically.

**Comment by UbaserB** [comment]
I got something coming up that will take a bunch of my time so im drafting it for now

**Comment by UbaserB** [comment]
Turns out you cant draft on mobile so I’ll wait until tomorrow morning unless someone does it first.

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:48]: write this in BaseCrowbar and remove duplication from all other

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:51]: same as above

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:91]: name same as in BaseCrowbar, can be removed

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:107]: remove as above

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:123]: remove as above

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml]: Not applicable here, it breaks things.

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml]: Also not applicable here.

**Comment by UbaserB** [comment]
Finally completed and ready for review

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:86]: you should write `size: Small` for it to fit in pockets (as said in desc)

**Review by lzk228 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:86]: i thought they said it was okay

File: Resources/Prototypes/Entities/Objects/Tools/crowbars.yml
Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Tools/crowbars.yml:66]: its there for a reason, im not touching it.

**Comment by lzk228** [comment]
hey!

**Comment by UbaserB** [comment]
> hey!

wait lol that's pretty funny it got merged without the reviews being addressed

**Pull request merged**
